XDMA/linux-kernel/xdma/libxdma.c

```diff
@@ -3530,6 +3533,30 @@ ssize_t xdma_xfer_submit(void *dev_hndl, int channel, bool write, u64 ep_addr,
 
 		switch (xfer->state) {
 		case TRANSFER_STATE_COMPLETED:
+			if (!write) {
```

Comment:
It means _read_ right?

Reply:
Yes if the write variable is set to 0, it means read operation